ee/tasks/subscriptions/__init__.py

```diff
@@ -144,12 +145,15 @@ def schedule_all_subscriptions() -> None:
         deliver_subscription_report.delay(subscription.id)
 
 
-@app.task()
+report_timeout_seconds = settings.PARALLEL_ASSET_GENERATION_MAX_TIMEOUT_MINUTES * 60 * 1.5
```

Comment:
not sure why this is `* 60 * 1.5` when the setting is `PARALLEL_ASSET_GENERATION_MAX_TIMEOUT_MINUTES` in minutes. 🤔